robottelo/test.py

```diff
@@ -469,9 +488,18 @@ def _join_all_threads(self, thread_list):
         for thread in thread_list:
             thread.join()
 
+    def _get_output_filename(self, file_name):
+        """Get type of test: ak/att/del/reg as output file name"""
```

Comment:
Please expand this docstring. What is `file_name`? Is it a file path that will be opened with `open`? Is it a plain string name that will be parsed? What will be returned?